lib/matplotlib/axis.py

```diff
@@ -1023,6 +1023,76 @@ def set_default_intervals(self):
         # attribute, and the derived code below will check for that
         # and use it if it's available (else just use 0..1)
 
+    def _set_lim(self, v0, v1, *, emit=True, auto):
+        """
+        Set view limits.
+
+        This method is a helper for the Axes ``set_xlim``, ``set_ylim``, and
+        ``set_zlim`` methods.  This docstring uses names corresponding to
+        ``set_xlim`` for simplicity.
```

Comment:
```suggestion
        ``set_zlim`` methods.
```
This is no longer the case.

Reply:
Oh, the docstring (and signature) don't have that notion, but some comments in the function still have.